packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -272,6 +272,14 @@ export interface IContextualMenuListProps {
   totalItemCount: number;
   hasCheckmarks: boolean;
   hasIcons: boolean;
+  defaultMenuItemRenderer: (
+    item: IContextualMenuItem,
+    index: number,
+    focusableElementIndex: number,
+    totalItemCount: number,
+    hasCheckmarks: boolean,
+    hasIcons: boolean
+  ) => React.ReactNode;
```

Comment:
I'm not 100% on this but I think that this return type should be `React.ReactElement | null` since that's the same type that `React.FunctionComponent` returns

Reply:
I think it needs to be a ReactNode based on this: https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement 

Since we're returning an array of ReactNodes, only ReactNode can represent this. When we call this function in ContextualMenu's render function, that will end up being a ReactElement since there's only a single parent.